src/components/tabs/tabs.js

```diff
@@ -195,7 +197,12 @@ export default Vue.extend({
       type: [String, Array, Object],
       default: null
     },
-    activeClass: {
```

Comment:
If removing a prop... it is better to deprecate it, and merge it's value into one of the new props.

Otherwise it becomes a breaking change.

Reply:
Or was `activeClass` a new prop that you had added?

Reply:
Yeah, it was a new one.

Reply:
OK cool